task/aws/task.go

```diff
@@ -92,139 +93,139 @@ type Task struct {
 }
 
 func (t *Task) Create(ctx context.Context) error {
-	log.Println("[INFO] Creating DefaultVPC...")
+	logrus.Debug("Creating DefaultVPC...")
```

Comment:
are these deliberately changed from `.Info` to `.Debug`?

Reply:
Yes. I do not think that they should not be Info level. However I might be totally wrong

Reply:
I believe that they are useful as part of the `INFO` logging level, as they provide valuable insights on the progress of creation and deletion operations.

Still, update (refresh) operations [run several times](https://github.com/iterative/terraform-provider-iterative/pull/423#issuecomment-1058574085) and add a bit of extra noise. Is this the reason why you prefer logging all these operations with the `DEBUG` level? 🤔

Reply:
> Still, update (refresh) operations https://github.com/iterative/terraform-provider-iterative/pull/423#issuecomment-1058574085 and add a bit of extra noise.

Exactly that. At the time we fix it would be nice to put it on INFO level but right now is not clear for the user why its repeated

Reply:
The last one in this bunch could be info, i.e.

```diff
-logrus.Debug("Done!")
+logrus.Info("Done")
```

Reply:
And parhaps replaced by `{operation} done` where `{operation}` is ´create´ or `read` or `delete`

Reply:
> The last one or two in this bunch could be info, i.e.

Even repeating it three times? Could not it be a bit misleading?

Reply:
Yes, it would still be a bit confusing. It would be better to fix the cause of this issue. I'll look into it later; it's relatively easy to cache reads.

Reply:
Tracked on https://github.com/iterative/terraform-provider-iterative/issues/290#issue-1062608705

Reply:
Still, I'm tempted to switch them back to `INFO` even if there are repeated reads. That information is very useful to debug some timeout issues.